frontend/package.json

```diff
@@ -21,23 +21,23 @@
     "build:analyze": "export _ODH_OUTPUT_ONLY=true; run-s build build:bundle-profile build:bundle-analyze",
     "build:bundle-profile": "webpack --config ./config/webpack.prod.js --profile --json > ./bundle.stats.json",
     "build:bundle-analyze": "webpack-bundle-analyzer ./bundle.stats.json",
-    "build:storybook": "build-storybook -s ./config",
+    "build:storybook": "storybook build -s ./config",
     "build:clean": "rimraf ./public",
     "build:prod": "webpack --config ./config/webpack.prod.js",
     "format": "prettier --write \"./**/*.ts\" \"./**/*.tsx\" \"./**/*.scss\"",
     "start:dev": "webpack serve --hot --color --config ./config/webpack.dev.js",
-    "test": "run-s test:lint test:jest test:type-check",
-    "test:jest": "jest",
-    "test:ci": "export CI=true; jest --collectCoverage",
-    "test:clearCache": "jest --clearCache",
-    "test:dev": "run-s test:lint test:local",
+    "test": "run-s test:lint test:unit test:type-check test:integration-ci",
+    "test:dev": "run-s test:lint test:unit test:accessibility test:integration",
     "test:fix": "eslint --ext .json,.js,.ts,.jsx,.tsx ./src --fix",
     "test:lint": "eslint --max-warnings 0 --ext .json,.js,.ts,.jsx,.tsx ./src",
-    "test:local": "jest --watchAll",
     "test:type-check": "tsc --noEmit",
-    "test:storybook": "test-storybook --verbose",
-    "test:storybook-ci": "npx concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test:storybook\"",
-    "storybook": "start-storybook -p 6006 -s ./config"
+    "test:jest": "jest",
+    "test:unit": "npm run test:jest",
+    "test:accessibility": "test-storybook --verbose",
+    "test:integration": "playwright test ./src/__tests__/integration",
+    "test:integration-ci": "npx concurrently -k -s first -n \"SB,TEST\" \"npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test:accessibility && npm run test:integration\"",
+    "test:e2e": "export E2E=1; playwright test ./src/__tests__/e2e",
+    "storybook": "storybook dev -p 6006 -s ../config"
```

Comment:
Aside from serving up the infra for other tests, does this have any value anymore?

Reply:
i think so. it is good for test any a11y errors and making sure all the mocks are working correctly